hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -1093,12 +1101,48 @@
 </property>
 
 <property>
-  <name>fs.s3a.fast.upload</name>
+  <name>fs.s3a.block.output</name>
```

Comment:
afraid so, that bit had missed the push as I forgot to --force the patch up at the end of the day. Will push it up with all the comments here, after another test run